src/main.cpp

```diff
@@ -28,6 +52,22 @@ int main(int argc, char *argv[])
     });
     parser.process(app);
 
+    QSettings settings;
+    QFile file(settings.fileName());
+    qDebug() << "Config file: " << file.fileName();
+    if (!file.exists()) {
+        qDebug() << "WARNING: Settings file " << file.fileName() << "not found!";
+        qDebug() << "Writing new settings file with default values.";
+        WriteDefaultSettings();
+    }
+
+    QVariantList pages;
+    int size = settings.beginReadArray("enabled_pages");
```

Comment:
```suggestion
    int size = settings.beginReadArray("enabled-pages");
```

Reply:
Why use a different array name than in qt5? Shouldn't we unify the settings file vars? This might be more error-prone.

Reply:
There is no need for unifying the settings files, as qtphy is a different program and we don't aim to provide any backwards compatibility with the Qt5 demo. It is much more important to be consistent in qtphy's source code.